lib/Runtime/Library/TypedArray.cpp

```diff
@@ -2815,28 +2562,41 @@ namespace Js
         return Js::JavascriptNumber::ToVarNoCheck(currentRes, scriptContext);
     }
 
+    // static 
+    TypedArrayBase * TypedArrayBase::ValidateTypedArray(Arguments &args, ScriptContext *scriptContext, LPCWSTR apiName)
```

Comment:
>ValidateTypedArray [](start = 37, length = 18)

nit: instead of making this an overload of ValidateTypedArray(), I would call it "ValidateThisAsTypedArray" or something like that, to be more descriptive.